src/server/scripts/Outland/HellfireCitadel/HellfireRamparts/boss_vazruden_the_herald.cpp

```diff
@@ -436,6 +439,10 @@ class boss_vazruden_the_herald : public CreatureScript
                                 return;
                             }
                         }
+			if (!(Nazan && Nazan->IsAlive()) && !(Vazruden && Vazruden->IsAlive()))
+			{
+				me->DisappearAndDie();
+			}
```

Comment:
fix indentation

Reply:
(1 tab = 4 blank spaces, otherwise GitHub will show the tab width = 8 spaces)